apix-integrationtests/build.gradle

```diff
@@ -83,14 +85,16 @@ subprojects {
         dependsOn composeUpTask
         doFirst {
```

Comment:
Instead of having it as a `doFirst` here, it is maybe better to have it as a `doLast` block on your `composeUpTask`?

Then it does not matter which task get executed first when doing the integration tests